src/sth_database.js

```diff
@@ -355,7 +416,7 @@
    * @param {Function} callback Callback to inform about any possible error or results
    */
   function getRawData(collection, entityId, entityType, attrName, lastN, hLimit, hOffset,
```

Comment:
Agreed and to be solved in a forthcoming PR including the refactoring of the whole app file structure to make it follow Telefónica R&D's Node.js application guidelines... ;) Thanks for pointing it out... ;)